v1/src/test/java/com/google/cloud/teleport/bigtable/AvroToBigtableLT.java

```diff
@@ -81,7 +83,7 @@ public void setup() throws IOException {
             gcsClient
                 .uploadArtifact(
                     "input/schema.json",
-                    Resources.getResource("AvroToBigtablePerformanceIT/schema.json").getPath())
+                    Resources.getResource("AvroToBigtableLT/schema.json").getPath())
```

Comment:
nit: I've been using `.createArtifact` and pasting the schema to have more DAMP/readable tests

Reply:
Good point. I think I have done this in multiple places, will change it together in a separate PR.